charts/core/src/core.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { storybookArgTypes } from '@lg-tools/storybook-utils';
+import { StoryFn } from '@storybook/react';
+
+import { LineProps } from './Line';
+import { makeLineData } from './testUtils';
+import { Chart, Grid, Line } from '.';
+
+export default {
+  title: 'Charts/Core',
+  component: Chart,
+  argTypes: {
+    darkMode: storybookArgTypes.darkMode,
+    data: {
+      control: {
+        type: 'object',
+      },
+    },
+    verticalGridLines: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    horizontalGridLines: {
+      control: {
+        type: 'boolean',
+      },
+    },
+  },
+  decorator: (Instance, context) => {
+    return <Instance darkMode={context?.args.darkMode} />;
+  },
+};
+
+interface StoryChartProps {
+  data: Array<LineProps>;
+  verticalGridLines: boolean;
+  horizontalGridLines: boolean;
+}
+
+const Template: React.FC<StoryChartProps> = props => {
```

Comment:
Was `StoryFn` not working here?

Reply:
`StoryFn<StoryChartProps>` wasn't working for some reason, it was leading to the following linting error:
```
/Users/terrence/Code/leafygreen-ui/charts/core/src/core.stories.tsx
  42:11  error  'data' is missing in props validation                 react/prop-types
  42:17  error  'verticalGridLines' is missing in props validation    react/prop-types
  42:36  error  'horizontalGridLines' is missing in props validation  react/prop-types
  46:13  error  'data.map' is missing in props validation             react/prop-types
```